### ISSUE TITLE
New preset: Bike repair shop

### ISSUE THREAD

**Issue by forteller** [openstreetmap/id-tagging-schema#43]
I feel like it's unnecessarily hard right now to add a bike repair shop that only repairs bikes, and doesn't sell any. You have to know that you open up the tags manager and add service:bicycle:retail no, after adding a node for a bicycle shop. 

But if you want to add a repair shop, you won't search for a bike shop, and might not even find the correct node to add at all. This might be worse in other languages. In english the word "shop" is included in both words, but for example in Norwegian a bike shop (selling bikes) is "sykkelbutikk", while a bicycle repair shop is "sykkelreparatør" or "sykkelverksted". So if you only see the preset "sykkelbutikk" you'll conclude that you can't add a "sykkelreparatør/verksted", and give up.

So I propose a new preset with shop=bicycle, service:bicycle:repair=yes and service:bicycle:retail=no

Thank you!

**Comment by westnordost** [comment]
Can you usually buy bike parts at a _sykkelverksted_?

**Comment by forteller** [comment]
Sure, mostly, I guess. A few spare parts. Still, if I see "bikeshop" on the map I think you can actually buy bikes there.

**Comment by maro-21** [comment]
> Still, if I see "bikeshop" on the map I think you can actually buy bikes there.

You will still see bikeshop icon on the map even after adding this preset because all of them are `shop=bicycle`.

**Comment by forteller** [comment]
Hi. I would like to ask again if there's any chance of this being added. Thanks!

**Comment by tordans** [comment]
@forteller Please have a look at our new guidelines which help us form an opinion on the "if". 

One thing would be to research the usage numbers for the proposals tag combination and if the are any any effects with other combinations. 

> shop=bicycle
> service:bicycle:repair=yes
> service:bicycle:retail=no

**Comment by Coehill** [comment]
They could be differentiated by a bicycle icon with a little wrench next to it.

**Comment by Coehill** [comment]
> @forteller Please have a look at our new guidelines which help us form an opinion on the "if".
> 
> One thing would be to research the usage numbers for the proposals tag combination and if the are any any effects with other combinations.
> 
> > shop=bicycle
> > service:bicycle:repair=yes
> > service:bicycle:retail=no

I am seeing 522 nodes and 46 polygons with this tag combination globally. Keep in mind that many of the shop=bicycle nodes don't include _any_ sort of service tags since they are not fully mapped.
Overpass API Query:
```
/*
Zoom out all the way and hit run.
*/
[out:json][timeout:120];
// gather results
nwr["shop"="bicycle"]["service:bicycle:repair"="yes"]["service:bicycle:retail"="no"]({{bbox}});
// print results
out geom;
```

Bicycle shops that primarily do repair and do not do retail sales are in the minority, but they definitely exist.  Often, it is small, independent organizations.  IMO, it does makes sense to include this tag.  It's similar how there are car dealers which offer both sales and repairs, and there are auto mechanics which only offer repairs.  Similarly, there are bike shops which do sales and repairs vs. bike repair shops.

**Comment by forteller** [comment]
>  It's similar how there are car dealers which offer both sales and repairs, and there are auto mechanics which only offer repairs. Similarly, there are bike shops which do sales and repairs vs. bike repair shops.

Exactly! And if you want to add a car mechanic to OSM, there's a Car Repair Shop preset in iD, while if you want to add a place that sells cars there's a Car Dealership preset. You don't need to know to go in and tweak the presets to set the different types of places when it comes to cars. You only have to do this for places related to bikes. 

I think that's unreasonable, and unfairly prioritizes mapping car related places.
